<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tw_orders">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Orders</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>-10</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Trade Signal Strategy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Strategy Summary</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_summary">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="autoScrollMargin">
    <number>0</number>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>0</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>110</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>0</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>24</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string>Total Sell Orders</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>Total Buy Orders</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>Total Profit</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Main</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
